### ISSUE TITLE
Add MVS SPIE macro support

### ISSUE THREAD

**Issue by adelosa** [z390development/z390#420]
https://www.ibm.com/docs/en/zos/2.5.0?topic=sspie-description

Looks to be similar to ESPIE, but 24 bit version. Maybe some kind of wrapper/alias for existing ESPIE macro would work here.

Required to support assembly of existing assembler mainframe apps.

**Comment by abekornelis** [comment]
You are correct, SPIE is a precursor of the ESPIE. We'll have to do some investigation, yet with ESPIE in place, implementing SPIE should be feasible. I hope I'm right.
